app/matplotlib.ipynb

```diff
@@ -86,23 +105,51 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
     "x = np.random.rand(50)\n",
     "y = np.random.rand(50)\n",
     "colors = np.random.rand(50)\n",
     "sizes = 100 * np.random.rand(50)\n",
-    "\n",
-    "# write your code here"
+    "df1 = pd.DataFrame({\n",
+    "    'x': x,\n",
+    "    'y': y,\n",
+    "    'size': sizes,\n",
+    "    'color': colors\n",
+    "})\n",
+    "plt.scatter(df1['x'], df1['y'], s=sizes, c=colors)\n",
+    "plt.xlabel('X')\n",
+    "plt.ylabel('Y')"
```

Comment:
The scatter plot is missing a colorbar label and a title. You can add a colorbar with a label using `plt.colorbar(label='Colorbar Label')` and set a title using `plt.title('Your Title Here')`.